scripts/travis/mode_tests-default.sh

```diff
@@ -41,6 +41,13 @@ if command -v docker > /dev/null ; then
   export LISK_ENABLED=1
   fold_end
 
+  fold_start "cosmos-start"
+  mkdir -p ./scripts/cosmos/.gaiad/data
+  cp ./scripts/cosmos/priv_validator_state.template.json ./scripts/cosmos/.gaiad/data/priv_validator_state.json
```

Comment:
Can't we move those commands into `./scripts/cosmos/start.sh`? This looks very inconvenient to start manually

Reply:
Overwriting the `priv_validator_state.json` file will effectively reset the blockchain.

Reply:
Which is what the other start scripts do as well. No other `./stop.sh && ./start.sh` would pickup the previous blockchain state

Reply:
Good point.